src/js/components/picklist/index.js

```diff
@@ -107,7 +108,9 @@ const Picklist = CollectionView.extend({
   className: 'picklist__group',
   tagName: 'li',
   template: hbs`
-    {{#if headingText}}<div class="picklist__heading">{{ headingText }}</div>{{/if}}
+    {{#if headingText}}
+      <div class={{ headingClassName }}>{{ headingText }}</div>
+    {{/if}}
```

Comment:
This allows me to apply specific styling to the headings in the top nav menu, without effecting other picklists. For the top nav menu, we want those headings to have `14px` instead of the normal `12px` used on other picklists.